packages/foreman/rubygem-redfish_client/rubygem-redfish_client.spec

```
@@ -76,11 +46,11 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%exclude %{gem_instdir}/.codeclimate.yml
+%{gem_instdir}/.codeclimate.yml
+%exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
 %exclude %{gem_instdir}/.rubocop.yml
-%exclude %{gem_instdir}/.simplecov
-%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/.simplecov
```

```suggestion
%exclude %{gem_instdir}/.simplecov
```

https://github.com/theforeman/foreman-packaging/pull/8371